io/disk/sysfsdisk_att.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2023 IBM
+# Author: Maram Srimannarayana Murthy <msmurthy@linux.vnet.ibm.com>
+
+"""
+Check all sysfs queue attributes by reading or writing different values
+"""
+
+import os
+from avocado import Test
+from avocado.utils import genio
+
+
+class SysfsDisk_Att(Test):
+
+    """
+    Checks all Sysfs_Queue attributes
+    """
+
+    def setUp(self):
+        """
+        Reading required arguments from yaml file
+        Setting up file paths
+        """
+        self.disk = self.params.get("disk", default="")
+        if not self.disk:
+            self.cancel("No disk input, please update yaml with disk")
+        self.disk = self.disk.split("/")[-1]
+        self.queue_param_disk_dir = os.path.join("/sys/block",
+                                                 self.disk, "queue")
+        self.param_name = self.params.get("param_name",
```

Comment:
`param_name` does not match yaml `attribute_name`  please keep in sync

Reply:
@abdhaleegit yeah missed this change when faced problem with branch header. Thank you for your catch. I have done required changes and pushed the change. Following us the output of execution run

[root@ltcever87-lp14 disk]# avocado run --max-parallel-tasks=1 sysfsdisk_att.py -m sysfsdisk_att.py.data/sysfsdisk_att.yaml 
JOB ID     : 2dc9dfd759eb79d8b5b5ce272edc92794f7aec68
JOB LOG    : /home/avocado-fvt-wrapper/results/job-2023-07-25T09.47-2dc9dfd/job.log
 (1/3) sysfsdisk_att.py:SysfsDisk_Att.test;run-queue_attributes-io_poll-b91d: STARTED
 (1/3) sysfsdisk_att.py:SysfsDisk_Att.test;run-queue_attributes-io_poll-b91d:  ERROR: [Errno 22] Invalid argument (0.03 s)
 (2/3) sysfsdisk_att.py:SysfsDisk_Att.test;run-queue_attributes-io_poll_delay-bcc8: STARTED
 (2/3) sysfsdisk_att.py:SysfsDisk_Att.test;run-queue_attributes-io_poll_delay-bcc8:  PASS (0.01 s)
 (3/3) sysfsdisk_att.py:SysfsDisk_Att.test;run-queue_attributes-iostats-0928: STARTED
 (3/3) sysfsdisk_att.py:SysfsDisk_Att.test;run-queue_attributes-iostats-0928:  PASS (0.01 s)
RESULTS    : PASS 2 | ERROR 1 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/avocado-fvt-wrapper/results/job-2023-07-25T09.47-2dc9dfd/results.html
JOB TIME   : 17.04 s